### PR TITLE
lua/decoders- recent- decoder that disallows old messages

### PR DESCRIPTION
**Pull request by nathanleiby**
Currently, hard-coded to only allow messages that have timestamps more
recent than 2 weeks ago.

### CODE REVIEW

**Comment by nathanleiby** [comment]
cc @burnsed

**Comment by burnsed** [comment]
one suggestion, but LGTM

**Comment by nathanleiby** [comment]
Added ability to configure `max_age_in_days` d104e5d

File: lua/decoders/recent.lua [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+--------------------------------
+--
+--  Public interface
+--
+--------------------------------
+
+function process_message()
+    local msg_time_seconds = math.floor(read_message("Timestamp") / 1e9)
+    local two_weeks_ago = os.time() - (config.max_age_in_days*24*60*60) -- 2 weeks, 7 days, 24 hours, 60 minutes, 60 seconds
```
Comment (jiancheung) [inline lua/decoders/recent.lua]: the variable name and comment should change.

**Comment by jiancheung** [comment]
this looks good to me!

**Pull request merged**